pages/upgrade.rst

```diff
@@ -2,8 +2,19 @@
 Upgrading Graylog
 *****************
 
+When upgrading from a previous version of Graylog you basically follow the previous used installation method using the new version numbers. 
+
+The following Upgrade notes should be read carefully before you start the upgrade process. Breaking changes and dependency upgrades are documented in those upgrade notes.
+
+You should always follow the version trail and not jump a few versions up leaving out one or more versions. The upgrade notes are always written coming from the stable release before.
+
 .. toctree::
    :titlesonly:
    :glob:
 
    upgrade/graylog-*
+
+
+The Virtual Machine Appliance (OVA) and Amazon Web Services (AMI) Version of Graylog use the Omnibus package. The upgrade documentation is :ref:`part of the graylog-ctl documentation <upgrade_graylog_omnibus>`.
+
+Should the previous installation include the Graylog repository of the current Graylog version, update the repository package to the target version and use the system tools to upgrade the package. For .rpm based systems :ref:`this update guide <operating_package_upgrade_rpm-yum-dnf>` and for .deb based systems :ref:`this update guide <operating_package_upgrade_DEB-APT>` should help.
```

Comment:
I would simplify this a bit:

> Should the current installation use operating system packages, update the repository package to the target version and use the system tools to upgrade the package.

Reply:
I had that before in such a simple way and notice that some users did not know how to do. Additional I want to link from that page to the actual page where the update is described.